plugins/jobtype/jobtypes/teradataToHdfs/plugin.properties

```diff
@@ -0,0 +1,5 @@
+mr.listener.visualizer=false
+
+#Below is to use latest avro jar for TDCH
```

Comment:
Can we avoid adding 
plugins/jobtype/jobtypes/hdfsToTeradata/lib/teradata-connector-1.4.1.jar
plugins/jobtype/jobtypes/teradataToHdfs/lib/teradata-connector-1.4.1.jar

as we already have extlib/teradata-connector-1.4.1.jar, which means we expect teradata-connector-1.4.1.jar to be in extlib for azkaban exec server deployment ?